README.md

```diff
@@ -68,13 +87,20 @@ $ sudo chmod +x /usr/local/bin/docker-compose
 ### Run Up
 Because we need to create tunnel interface, we need to use privileged container with root permission.
 ```bash
-$ git clone https://github.com/calee0219/free5gc-docker.git
+$ git clone https://github.com/free5gc/free5gc-compose.git
 $ cd free5gc-docker
+$ make base
 $ docker-compose build
 $ sudo docker-compose up # Recommand use with tmux to run in frontground
-$ sudo docker-compose up -d # Run in backbround if need
+$ sudo docker-compose up -d # Run in backbround if needed
 ```
 
+## How to use it: vagrant box
```

Comment:
I think this section can put to Pre-request as an option.

Either install gtp5g on host or using the vagrant file.

Reply:
Yes you can see it that way. That's why I've put it as a subsection of `How to use it: bare metal`